frontend/translations/en.po

```diff
@@ -3187,4 +3199,5 @@ msgid "workspace.updates.update"
 msgstr "Update"
 
 msgid "workspace.viewport.click-to-close-path"
-msgstr "Click to close the path"
\ No newline at end of file
+msgstr "Click to close the path"
```

Comment:
You should also create the texts in spanish at least.

Reply:
The spanish texts, can be handled and is usually handled directly in weblate. So, unless we have spanish texts on designs, I think is better leave it to be translated later...